hummingbot/cli/settings.py

```diff
@@ -323,12 +333,53 @@ def save_to_yml(yml_path: str, cm: Dict[str, ConfigVar]):
                                                   type_str="list",
                                                   required_if=lambda: False,
                                                   default=[
-                                                      ["^.+(USDT|USDC|USDS|DAI|PAX|TUSD)$", 1000],
+                                                      ["^.+(USDT|USDC|USDS|DAI|PAX|TUSD|USD)$", 1000],
                                                       ["^.+ETH$", 10],
                                                       ["^.+BTC$", 0.5],
                                                   ]),
 }
+discovery_config_map = {
+    "primary_market":                   ConfigVar(key="primary_market",
+                                                  prompt="Enter your first exchange name >>> ",
+                                                  validator=is_exchange,
+                                                  on_validated=lambda value: required_exchanges.append(value)),
+    "secondary_market":                 ConfigVar(key="secondary_market",
+                                                  prompt="Enter your second exchange name >>> ",
+                                                  validator=is_exchange,
+                                                  on_validated=lambda value: required_exchanges.append(value)),
 
+    "target_symbol_1":                  ConfigVar(key="target_symbol_1",
```

Comment:
This configuration only allows the opportunity discovery process to process one market pair at a time... this doesn't seem to make a lot of sense. I think we need a way for users to specify multiple market pairs at once between two different exchanges.

Reply:
This is a list configuration that allows multiple markets.
For example:
target_symbol_1:
- WETH-TUSD
- WETH-DAI
- TUSD-DAI

Reply:
In the config setup you would enter something like: ["WETH-TUSD", "WETH-DAI"]